assets/style.scss

```diff
@@ -0,0 +1,145 @@
+---
```

Comment:
Note that these two lines are required by Jekyll, as per https://jekyllrb.com/docs/assets/ and are the reason we need the `preprocess_command` in `.scss-lint.yml`